### PR TITLE
initial skeleton commit

### PR DESCRIPTION
**Pull request by Basharfrancis**
initial skeleton commit

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# http://editorconfig.org
```
Comment (jamie3) [inline .editorconfig]: Do we need this or should we just use `eslint` and `prettierrc`?

File: .yarnrc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+"@rafflebox-technologies-inc:registry" "https://npm.pkg.github.com"
```
Comment (jamie3) [inline .yarnrc]: Dont need this

File: README.md [hunk 3]
```diff
@@ -1,2 +1,14 @@
 # quick-start
-This repository is the baseline for all lambda repos we will build in the future
```
Comment (jamie3) [inline README.md:2]: Change `quick-start` to something like `lambda-typescript`. As `quick-start` is too generic.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  "engines": {
+    "node": "^14 || ^16"
+  },
+  "scripts": {
+    "clean": "rm -rf build/",
+    "build": "yarn clean && tsc",
+    "format": "prettier --write \"**/*.{ts,js,json,md,yml}\"",
+    "lint": "eslint \"**/*.{ts,js}\""
+  },
+  "husky": {
```
Comment (jamie3) [inline package.json]: Let's use the new husky way. It puts everything in a `.husky` folder.

File: src/index.ts [hunk 5]
```diff
@@ -0,0 +1,2 @@
+let message: string = 'Hello, World!'
+console.log(message)
```
Comment (jamie3) [inline src/index.ts]: This isn't lambda code.

File: tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "incremental": true,
+    "strictPropertyInitialization": false,
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "target": "es2019",
```
Comment (jamie3) [inline tsconfig.json]: I think this is an old version?

**Review by jamie3 — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -1,2 +1,14 @@
...
+
+`Hello Word` This repository is the baseline for all lambda repo's we will build in the future
+
+# test
+
+Add `& yarn test ` for the line underneath in order to run test when commit
+
+```
+"hooks": {
```
Comment (jamie3) [inline README.md]: Can remove this

File: package.json [hunk 8]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@rafflebox-technologies-inc/lambda-typescript",
+  "version": "1.0.0",
+  "main": "index.js",
+  "description": "A simple example of a quick start for a new project",
+  "license": "MIT",
+  "author": "Rafflebox",
+  "engines": {
+    "node": "^14 || ^16"
```
Comment (jamie3) [inline package.json]: Let's use node 16. I think Lambda works with node 16.

File: package.json [hunk 9]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@rafflebox-technologies-inc/lambda-typescript",
+  "version": "1.0.0",
+  "main": "index.js",
```
Comment (jamie3) [inline package.json:4]: I think this is `build/index.js`?

File: package.json [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+  "license": "MIT",
+  "author": "Rafflebox",
+  "engines": {
+    "node": "^14 || ^16"
+  },
+  "scripts": {
+    "clean": "rm -rf build/",
+    "build": "yarn clean && tsc",
+    "lint": "eslint \"**/*.{ts,js}\"",
+    "format": "prettier --write \"**/*.{ts,js,json,md,yml}\""
```
Comment (jamie3) [inline package.json]: We should have a script to package the lambda into a `.zip` file for uploading to aws.

File: package.json [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
```
Comment (jamie3) [inline package.json]: Remove this

**Review by jamie3 — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (Basharfrancis) [reply package.json]: I remove it for now

**Review by jamie3 — APPROVED** [review]

**Pull request merged**